client/src/actions/auth.js

```diff
@@ -37,33 +53,53 @@ export const signIn = (authData) => async(dispatch) => {
  * This action creator will be called when a user signs out.
  * @returns an action of type AUTH_SIGN_OUT.
  */
-export const signOut = () => async(dispatch) => {
-  try{
-    await api.logoutTeam()
-    dispatch ({
-      type: AUTH_SIGN_OUT
-    })
-
+export const signOut = () => async (dispatch) => {
+  try {
+    await api.logoutTeam();
     dispatch({
-      type: CLEAR_ERROR
-    })
+      type: AUTH_SIGN_OUT,
+    });
 
-  } catch(err){
+    dispatch({
+      type: CLEAR_ERROR,
+    });
+  } catch (err) {
     dispatch(errorActionGlobalCreator(err));
   }
 };
 
-export const authorizeJWT = () => async(dispatch) => {
-  try{
-    const { data } = await api.getTeamJWT()
+export const authorizeJWT = () => async (dispatch) => {
```

Comment:
Just one thing can improve performance, as both sidebar info and team info are needed to get from the backend for every refreshing, maybe we can just combine them to send only one API call instead of two separate calls? 
We can improve this in a separate ticket though.